ir/src/ir.rs

```diff
@@ -4,11 +4,11 @@ use serde::{Deserialize, Serialize};
 
 #[derive(Serialize, Deserialize, Debug, PartialEq)]
 pub struct Axis {
-    tag: String,
-    min: f32,
-    default: f32,
-    max: f32,
-    hidden: bool,
+    pub tag: String,
+    pub min: f32,
```

Comment:
Is `f32` the most appropriate type here? Genuinely a question.

Reply:
Good question. I don't want to block on or forget this so I've filed https://github.com/googlefonts/fontmake-rs/issues/6